RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -74,8 +75,9 @@ public static void setup(String apiKey,
         PlatformInfo platformInfo = new PlatformInfo(PLATFORM_NAME, PLUGIN_VERSION);
         Store store = useAmazon ? Store.AMAZON : Store.PLAY_STORE;
         DangerousSettings dangerousSettings = getDangerousSettingsFromJSON(dangerousSettingsJSON);
-        CommonKt.configure(UnityPlayer.currentActivity,
-                apiKey, appUserId, observerMode, platformInfo, store, dangerousSettings,
+        CommonKt.configure(UnityPlayer.currentActivity, apiKey, appUserId,
+                observerMode ? PurchasesAreCompletedBy.MY_APP : PurchasesAreCompletedBy.REVENUECAT,
```

Comment:
Might be worth extracting this to a variable, as we do for some other parameters like this?